Packs/CommonScripts/Scripts/DeleteContext/DeleteContext.js

```diff
@@ -134,9 +175,36 @@ if (shouldDeleteAll) {
     return "Successfully deleted index " + index + " from key " + args.key;
 
 } else {
-    var key = args.key;
-    if (isSubPlaybookKey) {
-      key = 'subplaybook-${currentPlaybookID}.' + key;
+    // Supporting comma separated list of keys to be deleted.
+    var keysToDelete = args.key.split(',')
```

Comment:
Is there a validation for args.key ?
args is an input from user / subplaybook right ?

What if key is a number / NaN / Object ?

Reply:
args.key is automatically casted to string.

numbers are casted to string:
![image](https://github.com/demisto/content/assets/158817412/fe5a0188-4d8b-4214-8bb5-51ef56726bdb)


empty value returns an error:
![image](https://github.com/demisto/content/assets/158817412/e16618b8-c923-4ef7-9590-57988dd67800)